test/test_coinbase_pro_user_stream_tracker.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+
+from os.path import join, realpath
+import sys; sys.path.insert(0, realpath(join(__file__, "../../")))
+
+import asyncio
+import logging
+import unittest
+import conf
+from typing import (
+    Optional
+)
+from wings.market.coinbase_pro_market import CoinbaseProAuth
+from wings.tracker.coinbase_pro_user_stream_tracker import CoinbaseProUserStreamTracker
+
+
+class CoinbaseProUserStreamTrackerUnitTest(unittest.TestCase):
+    user_stream_tracker: Optional[CoinbaseProUserStreamTracker] = None
+
+    @classmethod
+    def setUpClass(cls):
+        cls.ev_loop: asyncio.BaseEventLoop = asyncio.get_event_loop()
+        cls.coinbase_pro_auth = CoinbaseProAuth(conf.coinbase_pro_api_key,
+                                                conf.coinbase_pro_secret_key,
+                                                conf.coinbase_pro_passphrase)
+        cls.symbols = ["ETH-USDC"]
+        cls.user_stream_tracker: CoinbaseProUserStreamTracker = CoinbaseProUserStreamTracker(
+            coinbase_pro_auth=cls.coinbase_pro_auth, symbols=cls.symbols)
+        cls.user_stream_tracker_task: asyncio.Task = asyncio.ensure_future(cls.user_stream_tracker.start())
+
+    def test_user_stream(self):
+            # Wait process some msgs.
+            self.ev_loop.run_until_complete(asyncio.sleep(120.0))
+            print(self.user_stream_tracker.user_stream)
```

Comment:
is there a test you can add here?

Reply:
I'm copying the Binance user stream test. It's hard to test because not sure what message would come in next